segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -306,7 +311,14 @@ protected void doStart() {
         }
 
         // Delayed start. Secondary services need not be started in order for us to accept requests.
-        delayedStart.thenComposeAsync(v -> startSecondaryServicesAsync(), this.executor)
+        delayedStart
+                .thenComposeAsync( v -> {
+                    if (this.storage instanceof ChunkedSegmentStorage) {
+                        return ((ChunkedSegmentStorage) this.storage).finishBootstrap();
```

Comment:
This is because the storage unit tests can not depend on container code. 
All unit tests work at higher level of abstraction of `AbstractTaskQueue<TaskType>` and do not need to depend on actual underlying implementation.